paper.tex

```diff
@@ -78,9 +78,9 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{INTRODUCTION}
 
-In the advent of connected vehicles and autonomous transportation, equipping consumer vehicles with the necessary equipment to facilitate high-end automated collision avoidance systems can be prohibitively expensive.
-Technologies such as Radar, LIDAR and high-end cameras that are typically used for collision prediction are likely too expensive for most consumers.
-On the other hand, dash-cams, which are cheap video cameras that can be mounted to a car's dashboard, have become increasingly affordable and prevalent as a means to capture a driver’s perspective.
+In the advent of connected vehicles and autonomous transportation, we are equipping vehicles with the necessary equipment to facilitate high-end automated collision avoidance systems.
+Using these vehicles to simulate collision situations has a two-fold cost. Both the car and expensive / experimental sensors are put at risk.
```

Comment:
I would make the following change:

"Using these vehicles to simulate collision situations has a two-fold cost: both the car and expensive data-gathering equipment are put at risk."